rs3_trading/database/ge_tick_database.py

```diff
@@ -3,18 +3,16 @@
 
 
 def create_price_table(con: DuckDBPyConnection):
-    con.execute('create table if not exists rs3_price_data(time datetime, item varchar, item_name varchar, price long, primary key(item, time))')
+    con.execute('create table if not exists rs3_price_data(datetime_utc datetime, item varchar, item_name varchar, price long, primary key(item, datetime_utc))')
 
 
 def create_volume_table(con: DuckDBPyConnection):
-    con.execute('create table if not exists rs3_volume_data(time datetime, item varchar, item_name varchar, volume long, primary key(item, time))')
+    con.execute('create table if not exists rs3_volume_data(datetime_utc datetime, item varchar, item_name varchar, volume long, primary key(item, datetime_utc))')
 
 
 def insert_into_table(con: DuckDBPyConnection, table_name: str, column_names: list, dataframe: pd.DataFrame):
-    con.execute("""insert into {table_name}({column_names})
+    columns_str = ", ".join(column_names)
```

Comment:
It might be worth making a function to join with comma. You'll likely use it a lot. it belongs in utils